src/PluralResolver.js

```diff
@@ -111,7 +112,18 @@ class PluralResolver {
   getRule(code, options = {}) {
     if (this.shouldUseIntlApi()) {
       try {
-        return new Intl.PluralRules(getCleanedCode(code === 'dev' ? 'en' : code), { type: options.ordinal ? 'ordinal' : 'cardinal' });
+        const locales = getCleanedCode(code === 'dev' ? 'en' : code);
+        const type = options.ordinal ? 'ordinal' : 'cardinal';
+        const cacheKey = JSON.stringify({locales, type});
+        if (this.pluralRulesCache.has(cacheKey)) {
+          return this.pluralRulesCache.get(cacheKey);
+        }
+
+        const rule = new Intl.PluralRules(locales, {
+          type
+        });
```

Comment:
Tiny styling nit - can you change this to just `{ type }` since it should all fit on one line? That's a bit more readable